src/main/java/com/gargoylesoftware/htmlunit/html/DomNodeUtil.java

```diff
@@ -40,14 +40,14 @@
      * <p>
      * Calls {@link WebClientUtil#waitForJSExec(com.gargoylesoftware.htmlunit.WebClient)} before
      * executing the query.
-     * 
+     *
      * @param domNode the node to start searching from
      * @param xpathExpr the XPath expression
      * @return the list of objects found.
      */
     public static <E> List<E> selectNodes(final DomNode domNode, final String xpathExpr) {
         WebClientUtil.waitForJSExec(domNode.getPage().getWebClient());
-        return (List) XPathUtils.getByXPath(domNode, xpathExpr, null);
+        return (List) XPathHelper.getByXPath(domNode, xpathExpr, null, true);
```

Comment:
renamed in https://github.com/HtmlUnit/htmlunit/commit/30828a90cfab6bd087e43a468b7ece005a8f68d3#diff-47b6df4b439ad9746d4602e2dcc26927